avm/res/net-app/net-app-account/capacity-pool/volume/main.bicep

```diff
@@ -1,4 +1,5 @@
 metadata name = 'Azure NetApp Files Capacity Pool Volumes'
+
```

Comment:
```suggestion
```
Today these 2 lines are always clustered together. Doesn't matter at the end of the day, but it's one less change 😏